examples/2_read_serial.cpp

```diff
@@ -48,8 +48,18 @@ int main()
         cout << "\n\t" << m.first;
     cout << '\n';
     cout << "Iteration 100 contains " << i.particles.size() << " particle species:";
-    for( auto const& ps : i.particles )
+    for( auto const& ps : i.particles ){
         cout << "\n\t" << ps.first;
+        for( auto const& r : ps.second ){
+            cout << "\n\t" << r.first;
+            cout << '\n';
+        }
+    }
+
+    std::pair<std::string,openPMD::ParticleSpecies> s_e = *i.particles.begin();
+    std::pair<std::string,openPMD::Record> s_c = *s_e.second.begin();
+    auto charge = s_c.second[openPMD::RecordComponent::SCALAR].loadChunk<double>().get()[0];
```

Comment:
Maybe this return type is useful to know:
```suggestion
    std::shared_ptr< double > charge = s_c.second[openPMD::RecordComponent::SCALAR].loadChunk<double>();
```

Reply:
Thanks for changing this!